frontend/packages/webterminal-plugin/src/components/cloud-shell/WebTerminalConfiguration.tsx

```diff
@@ -256,7 +256,6 @@ const WebTerminalConfiguration: React.FC<{ readonly: boolean }> = ({ readonly })
           id="web-terminal-image"
           type="text"
           aria-label="web-terminal-image"
-          className="pf-v5-c-form-control"
```

Comment:
Styling is no longer needed?

Reply:
The styling is already applied since this is a `TextInput` react component